### PR TITLE
NEW Add update extension hooks for LinkFormFactory subclasses

### PR DESCRIPTION
**Pull request by robbieaverill**
Issue: https://github.com/silverstripe/silverstripe-cms/issues/2273

### CODE REVIEW

File: code/Forms/EditorEmailLinkFormFactory.php [hunk 1]
```diff
@@ -29,6 +29,8 @@ protected function getFormFields($controller, $name, $context)
             $fields->insertAfter('Link', TextField::create('Text', _t(__CLASS__.'.LINKTEXT', 'Link text')));
         }
 
+        $this->extend('updateFormFields', $actions, $controller, $name, $context);
```
Comment (ScopeyNZ) [inline code/Forms/EditorEmailLinkFormFactory.php]: Was this meant to be `$fields` instead of `$actions`?

**Comment by michelsteege** [comment]
Is there any news of a release date for this?

**Comment by robbieaverill** [comment]
Hi @michelsteege, we tend to release new minor versions every 3 months or so. The last was 4.3 (which this missed the cutoff for) so the next might be in late Feb/early March, although this message shouldn't be taken as a guarantee of that happening at that time

**Comment by michelsteege** [comment]
Thanks @robbieaverill  I wait patiently :-)

**Comment by robbieaverill** [comment]
@michelsteege if you really need to modify these classes, you can always extend them with Injector and modify the bits of logic affected in this patch (in the meantime)

**Pull request merged**